POLICIES.md

```diff
@@ -0,0 +1,58 @@
+# Introduction and Goals
+The ExaWorks SDK aims to support the workflows needs of a diverse range of users, administrators, and developers. It will enable teams to produce scalable and portable workflows for a wide range of exascale applications. The SDK will not replace the many workflow solutions already deployed and used by scientists, but rather it will provide a robust collection of community-identified technologies and components that can be leveraged by users.  Most importantly, the SDK will enable a sustainable software infrastructure for workflows so that the software artifacts produced by teams will be easier to port, modify, and utilize long after projects end. 
```

Comment:
Nit:
```suggestion
The ExaWorks SDK aims to support the workflow needs of a diverse range of users, administrators, and developers. It will enable teams to produce scalable and portable workflows for a wide range of exascale applications. The SDK will not replace the many workflow solutions already deployed and used by scientists, but rather it will provide a robust collection of community-identified technologies and components that can be leveraged by users.  Most importantly, the SDK will enable a sustainable software infrastructure for workflows so that the software artifacts produced by teams will be easier to port, modify, and utilize long after projects end. 
```